ansible/roles/layers-service/tasks/main.yml

```diff
@@ -0,0 +1,93 @@
+- include: ../../common/tasks/setfacts.yml
+
+- name: install Postgis and GDAL Execs
```

Comment:
i did ponder this. The postgis extensions are now shipped in one pkg.
They are also harmless if not used.